src/util/secrets/secrets.h

```diff
@@ -43,6 +43,13 @@
 #define DEFAULT_SEC_KCM_MAX_UID_SECRETS  64
 #define DEFAULT_SEC_KCM_MAX_PAYLOAD_SIZE 65536
 
+enum sss_sec_enctype {
+    SSS_SEC_PLAINTEXT,
+    SSS_SEC_MASTERKEY,
+    SSS_SEC_BASE64,
```

Comment:
What is the purpose of `SSS_SEC_BASE64`? This new "enc" type doesn't seem to be used anywhere...

Reply:
I tried this originally but it was still too slow so it is unused currently. Remove in a new patch due to conflicts during squash.